CMake/FindArrow.cmake

```diff
@@ -16,11 +16,19 @@ find_library(ARROW_LIB libarrow.a)
 find_library(PARQUET_LIB libparquet.a)
 find_library(ARROW_TESTING_LIB libarrow_testing.a)
 if("${ARROW_LIB}" STREQUAL "ARROW_LIB-NOTFOUND"
-   # OR "${PARQUET_LIB}" STREQUAL "PARQUET_LIB-NOTFOUND"
```

Comment:
@PHILO-HE can you remove this commented line?

Reply:
@majetideepak, just removed. Thanks!